src/Glpi/Form/Renderer/FormRenderer.php

```diff
@@ -85,7 +87,9 @@ public function render(Form $form): string
         // Load template
         $twig = TemplateRenderer::getInstance();
         $html .= $twig->render('pages/form_renderer.html.twig', [
-            'form' => $form,
+            'form'              => $form,
+            'is_anonymous_user' => FormAccessControlManager::getInstance()->isAnonymousForm($form)
+                && !Session::isAuthenticated(),
```

Comment:
```suggestion
            'is_anonymous_user' => !Session::isAuthenticated(),
```

More safe this way I guess (`isAnonymousForm` could be faulty and return a bad results, in the end we want to make sure that we known if the user is authenticated or not)